language/de/install.php

```diff
@@ -215,6 +214,14 @@
 	//
 	'EMAIL_CONFIG'	=> 'E-Mail-Konfiguration',
 
+	// Package info
+	'PACKAGE_VERSION'				=> 'Paketversion installiert',
+	'UPDATE_INCOMPLETE'				=> 'Deine phpBB-Installation wurde nicht richtig aktualisiert.',
+	'UPDATE_INCOMPLETE_MORE'		=> 'Bitte lies die folgenden Information, um diesen Fehler zu beheben.',
+	'UPDATE_INCOMPLETE_EXPLAIN'		=> '<h1>Unvollständige Aktualisierung</h1>
+
+		<p>Wir haben bemerkt, dass die letzte Aktualisierung deiner phpBB-Installation nicht vollständig durchgeführt wurde. Besuche die <a href="%1$s" title="%1$s">Datenbank-Aktualisierung</a>, stell sicher, dass <em>Nur die Datenbank aktualisieren</em> ausgewählt ist und klicke auf <strong>Absenden</strong>. Nachdem du die Datenbank erfolgreich aktualisiert hast, vergiss nicht das "install"-Verzeichnis zu löschen.</p>',
```

Comment:
Dazu können wir gerne in Ticket erstellen, es ist halt nicht zu sagen, ob "install" noch da ist oder schon weg. Nachdem Update kommt man immerhin auch erstmal ins ACP. Von daher würde ich da jetzt nicht ganze Sätze dazu dichten.